### PR TITLE
LL-388: Digital Identity Detail View

### PR DESCRIPTION
**Pull request by divyanshjain9**

### CODE REVIEW

File: ui/src/App.tsx [hunk 1]
```diff
@@ -131,6 +132,7 @@ function App() {
           <Route path="/issuers" element={<TrustedIssuersPage service={blockchainService} />} />
           <Route path="/identities" element={<IdentitiesPage service={blockchainService} />} />
           <Route path="/claims" element={<ClaimsPage service={blockchainService} />} />
+		  <Route path="/identities/id" element={<DigitalIdentityDetailView />} />
```
Comment (jkrinsky) [inline ui/src/App.tsx]: the path here should be identites/{selectedId}

File: ui/src/components/NavBar.jsx [hunk 2]
```diff
@@ -31,6 +31,7 @@ const NavBar = ({
           <ul className="space-y-2 mt-2">
             <li><Link to="/identities/add" className="hover:underline">Add new...</Link></li>
             <li><Link to="/identities" className="hover:underline">List</Link></li>
+            <li><Link to="/identities/id" className="hover:underline">Detail Page</Link></li>
```
Comment (jkrinsky) [inline ui/src/components/NavBar.jsx]: please update route as requested earlier

File: ui/src/components/DigitalIdentityDetailPage.jsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
+import { Button, Input } from 'antd'
+import React from 'react'
+
+function DigitalIdentityDetailView() {
+    const [displayName, setDisplayName] = React.useState('')
+
+    const firstCardData = [
```
Comment (jkrinsky) [inline ui/src/components/DigitalIdentityDetailPage.jsx]: Card data needs to be in an array

File: ui/src/components/DigitalIdentityDetailPage.jsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+                    </div>
+                </div>
+                <div className='border rounded-xl p-6 bg-white flex flex-col gap-3 '>
+                    <p>Investor Wallet Address</p>
+                    <p className='font-light text-4xl max-[500px]:text-base text-gray-300'>0x5555aaadwerrasdw553</p>
+                    <p>Investor KYC ID Provider Account Number</p>
+                    <p className='font-light text-4xl max-[500px]:text-base text-gray-300'>007</p>
+                </div>
+                <p>Active Claims</p>
+                <VioletCard data={firstCardData} />
```
Comment (jkrinsky) [inline ui/src/components/DigitalIdentityDetailPage.jsx]: Iterate through card data array here and create a card for each item

File: ui/src/components/DigitalIdentityDetailPage.jsx [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+                <VioletCard data={secondCardData} />
+            </div>
+            <div className='flex justify-end max-[500px]:justify-center'>
+                <Button className='max-[500px]:w-[50%] rounded-none my-6 mr-6 h-11 px-10 bg-[#9952b3] text-white'>Back</Button>
+            </div>
+        </div>
+    )
+}
+
+export const VioletCard = ({ data }) => {
```
Comment (jkrinsky) [inline ui/src/components/DigitalIdentityDetailPage.jsx]: rename to "ClaimCard"

**Review by jkrinsky — CHANGES_REQUESTED** [review]
Please see comments and make updates accordingly. Thank you!

File: ui/src/components/DigitalIdentityDetailPage.jsx (hunk 5 above)
Comment (divyanshjain9) [reply ui/src/components/DigitalIdentityDetailPage.jsx]: Done

File: ui/src/App.tsx (hunk 1 above)
Comment (divyanshjain9) [reply ui/src/App.tsx]: Done

File: ui/src/components/NavBar.jsx (hunk 2 above)
Comment (divyanshjain9) [reply ui/src/components/NavBar.jsx]: Done

File: ui/src/components/DigitalIdentityDetailPage.jsx (hunk 3 above)
Comment (divyanshjain9) [reply ui/src/components/DigitalIdentityDetailPage.jsx]: Done

File: ui/src/components/DigitalIdentityDetailPage.jsx (hunk 4 above)
Comment (divyanshjain9) [reply ui/src/components/DigitalIdentityDetailPage.jsx]: Done

**Comment by divyanshjain9** [comment]
@jkrinsky Kindly check the changes and confirm.

File: ui/src/components/DigitalIdentityDetailPage.jsx [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+                })}
+            </div>
+            <div className='flex justify-end max-[500px]:justify-center'>
+                <Button className='max-[500px]:w-[50%] rounded-none my-6 mr-6 h-11 px-10 bg-[#9952b3] text-white'>Back</Button>
+            </div>
+        </div>
+    )
+}
+
+export const ClaimCard = ({ data }) => {
```
Comment (jkrinsky) [inline ui/src/components/DigitalIdentityDetailPage.jsx]: why is this being exported? If it is being used in other pages, please make it it's own component. Also, can you move it above the main page component if it is staying local? I imagine we will use it in the edit confirmation view, too, so probably best to jut move it out into it's own component now.

**Review by jkrinsky — CHANGES_REQUESTED** [review]
please extract the claim card component into a separate file to be used by both the Digital Identity Detail view and the Edit Claim Confirmation view. Thank you!

File: ui/src/components/DigitalIdentityDetailPage.jsx (hunk 6 above)
Comment (divyanshjain9) [reply ui/src/components/DigitalIdentityDetailPage.jsx]: Updated

**Review by jkrinsky — APPROVED** [review]
LGTM!

**Pull request merged**